schema/postgres/1494804546_create_repositories_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE repositories (
+  id             UUID PRIMARY KEY                           DEFAULT gen_random_uuid(),
```

Comment:
> PostgreSQL automatically creates a unique index when a unique constraint or primary key is defined for a table. The index covers the columns that make up the primary key or unique constraint (a multicolumn index, if appropriate), and is the mechanism that enforces the constraint.

To answer your question (https://www.postgresql.org/docs/current/static/indexes-unique.html)